src/bionics.cpp

```diff
@@ -974,23 +975,43 @@ bool player::install_bionics( const itype &type, int skill_level )
     }
 
     const int pk = get_painkiller();
+    const int overall_pk_dur = ( get_effect_dur(effect_pkill1) + get_effect_dur(effect_pkill2) +
+                                 get_effect_dur(effect_pkill3) + get_effect_dur(effect_pkill_l) ) / MINUTES( 1 );
     int pain_cap = 100;
     if( has_trait( trait_PAINRESIST_TROGLO ) ) {
         pain_cap = pain_cap / 2;
     } else if( has_trait( trait_PAINRESIST ) ) {
         pain_cap = pain_cap / 1.5;
     }
 
+    int fa_level = get_skill_level( skill_firstaid );
+
+    if( has_trait( trait_PROF_MED ) ) {
+        fa_level =+5;
```

Comment:
Why is there a `+` in front of the `5`? `fa_level = 5` is the same, but shorter (and follows the established whitespace rules).